content/project-news/reactos-0415-released.md

```diff
@@ -0,0 +1,73 @@
+---
+title:       "ReactOS 0.4.15 released"
+author:      "ReactOS Team"
+date:        2025-03-01
+tags:        [ "release" ]
+banner:      "img/project-news/reactos-0415-released/thumb.png"
+---
+
+We are pleased to announce the release of ReactOS 0.4.15! There are some great new features including improvements to accessories such as notepad and paint, explorer shell, input method editor, registry healing, USB boot, Audio, memory management, ISA bus driver, and kernel debugger.
+
+This release is a culmination of the work of countless contributors since 0.4.14 was branched in 2020. This has been the largest release to date. There are nearly 8 times more commits this release than in 0.4.14. We are extremely proud of the progress we have made, and cannot wait to continue in this growth. Development on ReactOS has been growing exponentially! Let's dive in and see what's new.
+
+## Accessories and System Tools
+Katayama Hirofumi implemented quality of life improvements, performance enhancements, and new features such as the "text tool" in Paint and the "Now Printing" dialog in Notepad.
```

Comment:
Here, Katayama Hirofumi and Katayama Hirofumi MZ (katahiromz) are the same guy. Please unify notation.